Generator/Generator/MethodGen.swift

```diff
@@ -62,6 +62,22 @@ func isRefParameterOptional (className: String, method: String, arg: String) ->
     }
 }
 
+/// TODO: The current code generation for passing parameters is both inefficient, and technically unsafe. We don't need
+/// to use nested invocations of withUnsafePointer to generate pointers to multiple arguments, but we can instead generate
```

Comment:
Does the TODO here apply to this current PR, or the existing code?

I ask because this meads me to believe that this PR is "technically unsafe", I rather not merge code in that state.   Also curious, what is technical unsafe?

Reply:
I was talking about the pre-existing code. Storing the pointers to the arguments in _args, which is in a shallower stack frame.